pytripgui/app_logic/gui_executor.py

```diff
@@ -0,0 +1,60 @@
+from pytripgui.executor_vc.executor_view import ExecutorQtView
+
+from pytripgui.plan_executor.threaded_executor import ThreadedExecutor
+from pytripgui.tree_vc.TreeItems import SimulationResultItem
+from pytripgui.messages import InfoMessages
+
+from PyQt5.QtCore import QTimer
+
+
+class GuiExecutor:
+    def __init__(self, trip_config, patient, plan, result_callback, partnt_view):
+        if not plan.data.fields:
+            partnt_view.show_info(*InfoMessages["addOneField"])
+            return
+
+        self.result_callback = result_callback
+        self.done = False   # True when object can be removed from memory
+
+        self._gui_update_timer = QTimer()
+        self._thread = ThreadedExecutor(plan, patient, trip_config)
+
+        self._ui = ExecutorQtView(partnt_view)
+
+    def show(self):
+        self._ui.show()
+
+    def update_gui(self):
+        if self._thread.is_alive():
+            self._gui_update_timer.singleShot(10, self.update_gui)
```

Comment:
write a comment explaining what are the units of time here: 10 hours, 10 seconds ?